lua/rip-substitute/rg-operations.lua

```diff
@@ -17,6 +17,7 @@ local function runRipgrep(rgArgs)
 		config.regexOptions.pcre2 and "--pcre2" or "--no-pcre2",
 		"--" .. config.regexOptions.casing,
```

Comment:
I think to keep things consistent, this config option for case sensitivity should also be adapted. The config has 3 options which should be turned into a boolean to match the toggling here. 

Also, it should probably be renamed to match the one for starting with fixed strings. 

(And then this line 18 can be removed, as line 20 manages this already. )

Reply:
I think you can pretty much do what you did in the other PR, but for casing with the "startsWith" variable.